src/style.css

```diff
@@ -2,28 +2,37 @@
   box-sizing: border-box;
   margin: 0;
   border: 0;
-  padding-left: 20px;
+  padding: 0;
   list-style: none;
 }
 
-.leaderboard {
-  width: 80%;
-  align-items: stretch;
-  justify-content: space-between;
+body {
+  background-color: rgb(196, 244, 244);
+}
+
+h1 {
+  font-weight: 800;
+  font-size: 54px;
+  padding-left: 50px;
+  margin-bottom: 20px;
+  color: rgb(29, 25, 248);
+  background-color: rgb(174, 176, 176);
 }
 
 .main {
   display: grid;
-  grid-template-columns: 2fr 2fr;
-  align-items: center;
-  gap: 25px;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 100px;
```

Comment:
- [ ] Kindly try to stick to the provided wireframe, especially on your form section

![image](https://github.com/supreeti/Leaderboard/assets/98453979/f0e80bfe-a76d-4f80-93e3-506362a5329a)

### Checkout the provided wireframe

![image](https://github.com/supreeti/Leaderboard/assets/98453979/51eda352-b53f-4277-a818-a51bfb5d644e)

Reply:
change style as per wireframe.